### PR TITLE
.pullapprove.yml: fix array def

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
@vbatts @caniszczyk Could you please merge this as we don't have powers to force merge? We need this in order to unblock other PRs

**Comment by caniszczyk** [comment]
LGTM

File: .pullapprove.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+approve_regex: ^LGTM
 reject_regex: ^Rejected
 reset_on_push: true
+author_approval: ignored
 signed_off_by:
   required: true
 reviewers:
   teams:
-  - go-selinux-maintainers
+    - go-selinux-maintainers
```
Comment (runcom) [inline .pullapprove.yml]: Ops this is the wrong one again!

Comment (runcom) [reply .pullapprove.yml]: @caniszczyk

Comment (runcom) [reply .pullapprove.yml]: Nevermind

**Pull request merged**